newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js

```diff
@@ -229,32 +248,43 @@ export default class RenderedCustomObjectInstance extends RenderedInstance {
     const centerX = defaultWidth / 2;
     const centerY = defaultHeight / 2;
 
+    const width = this._instance.hasCustomSize()
+      ? this._instance.getCustomWidth()
+      : this.getDefaultWidth();
+    const height = this._instance.hasCustomSize()
+      ? this._instance.getCustomHeight()
+      : this.getDefaultHeight();
+
     for (
       let index = 0;
       index < this.childrenRenderedInstances.length;
       index++
     ) {
       const renderedInstance = this.childrenRenderedInstances[index];
       const childInstance = this.childrenInstances[index];
-      childInstance.x = (defaultWidth - renderedInstance.getDefaultWidth()) / 2;
```

Comment:
For now we can do like this, but in the future we could probably add a setting to choose for each child object instances if they are stretched or centered when rendered in the scene editor.
Could help for some basic cases like buttons or a few other simple objects.

Reply:
This should probably be the default layout when there is no `initialInstances` defined.

Reply:
Ok!